src/components/Layout/Collapsible/Collapsible.js

```diff
@@ -1,14 +1,21 @@
 import PropTypes from 'prop-types'
-import { useState } from 'react'
+import { useState, useEffect } from 'react'
 
 const Collapsible = ({
   heading,
   children,
   contentMargin,
   collapsableDivClassName,
+  onTasksAndSorsTab,
```

Comment:
This is functionally fine but it's a good practice for us to keep more generic components from knowing about our business logic as much as possible. In this case, knowing about what tab it's on via its prop name. All of the other props are really general names here and we should try to keep it like that :)

The underlying functionality we want here is for the parent to tell the collapsible whether to start open or closed. Any parent which doesn't specify this should get the existing open-by-default behaviour.

We can use a `startsClosed` prop with a more generic name and supply that as the initial state instead of using an effect hook to clean things up a little. If it's not passed in from the parent, the collapsible component will have the default open behaviour.

Please check out the `App.js` and `Collapsible.js` files in the sandbox here https://codesandbox.io/s/reverent-hermann-5gmw3?file=/src/Collapsible.js